packages/manager/apps/telecom/src/components/telecom/spare/sidebar/spare-sidebar.service.js

```diff
@@ -0,0 +1,27 @@
+angular.module('managerApp').service('SpareSidebar', class SpareSidebar {
+  /* @ngInject */
+  constructor($translate, SidebarMenu, OvhApiMeVipStatus) {
+    this.$translate = $translate;
+    this.SidebarMenu = SidebarMenu;
+    this.OvhApiMeVipStatus = OvhApiMeVipStatus;
```

Comment:
Isn't the VIP status already fetched at the root of the app ? Maybe we could try to get it in only one call if possible

Reply:
Yes it's a good idea. For the moment, it's called each time we need to know if the user is a VIP. It's possible that I need some help on this part.

Reply:
Hmm ok @marie-j , the `vipStatus` fetched at the root of the app, is located into telecom part. And so the related resolve is called when you entered into telephony section.

Spare management is located at the same level than `telecom-dashboard`(in term of application architecture), so the initialization through this service may be called, before any attempt to go through telecom part (and then, before having resolution of `vipStatus`).

Maybe you should ask to @jleveugle if something is possible, but if I'm not wrong, all parts of Manager Telecom will be split later, and so, fetch this vip status will be required for each of those parts.